packages/core/src/components/Table/Table.tsx

```diff
@@ -202,6 +203,7 @@ export interface TableProps<T extends object = {}>
   subtitle?: string;
   filters?: TableFilter[];
   initialState?: TableState;
+  emptyComponent?: JSX.Element;
```

Comment:
Should prolly be `emptyContent` instead as it isn't a component, could maybe be a `ReactNode` instead of `JSX.Element` as well